tests/integration/server/location/location.controller.test.ts

```diff
@@ -401,4 +402,77 @@ describe('Location API', function () {
       });
     });
   });
+  describe('GET /rooms/admin', function () {
+    let response: Response;
+    context('As an unauthenticated user', function () {
+      beforeEach(function () {
+        authStub.resolves(null);
+      });
+      it('should return a 400 status', async function () {
+        authStub.rejects(new ForbiddenException());
+
+        response = await request(api)
+          .get('/api/rooms/admin');
+
+        strictEqual(response.ok, false);
+        strictEqual(response.status, HttpStatus.FORBIDDEN);
+      });
+    });
+    context('As an authenticated user', function () {
+      let result: RoomAdminResponse[];
+      beforeEach(async function () {
+        authStub.resolves(adminUser);
+        response = await request(api)
+          .get('/api/rooms/admin');
+        result = response.body;
+      });
+      it('should return a 200 status', function () {
+        strictEqual(response.status, HttpStatus.OK);
+      });
+      it('should return a nonempty array of data', function () {
```

Comment:
Thinking about this further, I think that makes sense to remove, since the test below it is checking the rooms that are returned. This was removed in [61dfb74](https://github.com/seas-computing/course-planner/pull/570/commits/61dfb74ceae3646431b645fac4b31415ab9a5543).